### PR TITLE
[MU3] Fix typo in a the name of a style setting

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
For master see #7712

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
I'm slightly puzzled that 3.x needs these changes to vtests, but for master it does not?

File: libmscore/style.cpp [hunk 1]
```diff
@@ -434,7 +434,7 @@ static const StyleType styleTypes[] {
       { Sid::mmRestNumberPos,         "mmRestNumberPos",         Spatium(-1.5) },
       { Sid::hideEmptyStaves,         "hideEmptyStaves",         QVariant(false) },
       { Sid::dontHideStavesInFirstSystem,
-                                 "dontHidStavesInFirstSystm",    QVariant(true) },
+                                 "dontHideStavesInFirstSystem",  QVariant(true) },
```
Comment (njvdberg) [inline libmscore/style.cpp:437]: Does this name change cause compatibility issues with existing scores?

Comment (Jojo-Schmitz) [reply libmscore/style.cpp:437]: It may (read: will) cause downrev issues though.

Comment (njvdberg) [reply libmscore/style.cpp:437]: Ah, I missed that one.

**Pull request merged**